lib/websocket.js

```diff
@@ -100,6 +98,11 @@ function init (env, ctx, server) {
   function emitData (delta) {
     if (lastData.cals) {
       console.log(LOG_WS + 'running websocket.emitData', ctx.ddata.lastUpdated);
+      if (lastProfileSwitch != ctx.ddata.lastProfileFromSwitch) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2416943549/issues/source?bid=2452379&fileBranchId=3237267#l101)